web/tests/topic_list_data.test.js

```diff
@@ -250,6 +255,12 @@ test("get_list_info unreads", ({override}) => {
         return topic_name === "topic 4";
     });
 
+    add_unreads("topic 5", 1);
+    override(user_topics, "is_topic_unmuted", (stream_id, topic_name) => {
+        assert.equal(stream_id, general.stream_id);
+        return topic_name === "topic 5";
+    });
```

Comment:
This is fine for now but we should as a follow-up rewrite this test to just use the actual `user_topics` module and configure a set of data; this kind of hacky mock replacing a data module with a rule like this is generally to be avoided.